scripts/screener/screener.runner.ts

```diff
@@ -109,5 +107,13 @@ export async function screenerRunner(screenerConfigPath) {
       : undefined,
   });
 
-  await notifyIntegration(commit, checkUrl);
+  await notifyIntegration({ commit, url: checkUrl, status: 'in_progress', project: screenerConfig.projectRepo });
+}
+
+export async function cancelScreenerRun(screenerConfigPath) {
+  const screenerConfig: ScreenerRunnerConfig = require(screenerConfigPath) as any;
+  // https://github.com/microsoft/azure-pipelines-tasks/issues/9801
+  const commit = process.env.SYSTEM_PULLREQUEST_SOURCECOMMITID;
+
+  await notifyIntegration({ commit, url: '', status: 'completed', project: screenerConfig.projectRepo });
```

Comment:
I don't how it's implemented, but can we use `conclusion: skipped`? IMO it will be clearer what happened, for example: 
![image](https://user-images.githubusercontent.com/14183168/152982445-fb284254-3acb-4c74-85fa-0f02b0384703.png)


https://docs.github.com/en/rest/reference/checks#create-a-check-run

Reply:
I agree with this but would bring this up later in a new PR as a new feature as what's implemented here is the current behavior. Also because the proxy doesn't just propagate the status value into the Github API (I don't think) and that would required changes on the proxy code first, but correct me if I'm wrong @ling1726.

Reply:
@layershifter  it all depends on if our branch protection counts `skipped` as a valid status to merge the PR :)

Can we screenshare to check ? because I don't have access to branch protection

Reply:
@ling1726 as I know Github does not allow to configure statuses for checks in repository settings.

Reply:
so the lesson: 

- `cancelled` will not pass the check
- `skipped` will pass the check